proc/cut/cutter.go

```diff
@@ -20,7 +20,7 @@ type cutBuilder struct {
 func (cb *cutBuilder) cut(in *zng.Record) *zng.Record {
 	cb.builder.Reset()
 	for _, resolver := range cb.resolvers {
-		val := resolver(in)
+		val, _ := resolver.Eval(in)
```

Comment:
Should we at least add a panic on the error here?

Reply:
It was this way before my PR.  I will create a new issue for cut to send up errors.  There is another case that should go up.  It could also insert error values.  This is probably the best approach.

Reply:
Issue #1409